packages/gatsby-plugin-inferno/.gitignore

```diff
@@ -0,0 +1,2 @@
+/gatsby-node.js
+/gatsby-browser.js
```

Comment:
I think it should be something like:
```
/*.js
!index.js
yarn.lock
```
We don't want any `gatsby-` built file in git (including `gatsby-node`)

Reply:
I'll try to get to this tonight - thanks :)